SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -295,6 +295,26 @@ bool USpatialPackageMapClient::SerializeObject(FArchive& Ar, UClass* InClass, UO
 	return true;
 }
 
+const FClassInfo* USpatialPackageMapClient::TryResolveNewDynamicSubobjectAndGetClassInfo(UObject* Object, Worker_EntityId EntityId, USpatialClassInfoManager* ClassInfoManager)
+{
+	FUnrealObjectRef Ref = GetUnrealObjectRefFromObject(Object);
+	if (Ref.IsValid())
+	{
+		UE_LOG(LogSpatialPackageMap, Error, TEXT("Trying to resolve a dynamic subobject twice! Object %s, EntityId %d."), Object ? *Object->GetName() : TEXT("null"), EntityId);
+		return nullptr;
+	}
+
+	const FClassInfo* Info = ClassInfoManager->GetClassInfoForNewSubobject(Object, EntityId, this);
+
+	// If we don't get the info, an error is logged in the above function, that we have exceeded the maximum number of dynamic subobjects on the entity
+	if (Info)
+	{
+		ResolveSubobject(Object, FUnrealObjectRef(EntityId, Info->SchemaComponents[SCHEMA_Data]));
```

Comment:
Realise this isn't your change - but why don't we register owneronly and handover components also?

Reply:
Hmmm, I don't actually know... @Vatyx?

Reply:
Oh, because we don't generate them. :/